### PR TITLE
Unflatten data structure.

### PR DESCRIPTION
**Pull request by somexpert**
ping @ldko, this is up in dev right now.

### CODE REVIEW

**Comment by somexpert** [comment]
This will close #3

File: tests/test_utils.py [hunk 1]
```diff
@@ -87,6 +87,34 @@ def test_returns_correct_info(self, mock_getsize, mock_current_app, mock_decrypt
...
+                'fileset': '5',
+                'kind': 'captions',
+                'language': 'ger',
+            }
+        ]
+        mock_getsize.return_value = 256
+        result = get_files_info(pairpath, files)
+        assert result['1']['8']
+        assert result['2']['5']
+        assert not result['1']['1']
```
Comment (somexpert) [inline tests/test_utils.py]: eh, not sure this is useful. Guessing you agree with that sentiment @ldko

**Review by ldko — COMMENTED** [review]
These changes look ok to me. One thing I did notice in running this app, [you can end up with a double slash here](https://github.com/unt-libraries/aubrey-transcription/blob/dcbfa3e45d24dadc2a650627bbdbfde33b0f39b2/aubrey_transcription/utils.py#L50) if the transcription url ends in a slash. We could `rstrip` that or  something to prevent that.

File: tests/test_utils.py (hunk 1 above)
Comment (ldko) [reply tests/test_utils.py]: It doesn't bother me to have it there.

**Comment by somexpert** [comment]
hmmm, yeah, don't want that. I'll do what you suggest.

File: tests/test_utils.py (hunk 1 above)
Comment (somexpert) [reply tests/test_utils.py]: guess I'll leave it there then. for now...

**Comment by ldko** [comment]
ok :+1:

**Pull request merged**